Makefile

```diff
@@ -106,7 +106,7 @@ create-ns:
 # Run against the current locally configured Kubernetes cluster
 .PHONY: run
 run:
-	WATCH_NAMESPACE=$(WATCH_NAMESPACE) go run ./main.go controller --leader-elect=false
+	WATCH_NAMESPACE=$(WATCH_NAMESPACE) go run ./main.go controller --leader-elect=false --enable-operator-policy=true #--target-kubeconfig-path=./kubeconfig_managed2
```

Comment:
nit: remove commented part.

Adding `--enable-operator-policy=true` is a good idea though.